localstack/aws/handlers/__init__.py

```diff
@@ -13,6 +24,7 @@
 add_region_from_header = region.RegionContextEnricher()
 log_exception = logging.ExceptionLogger()
 log_response = logging.ResponseLogger()
+aggregate_response = response_aggregator.ResponseAggregator()
```

Comment:
could we maybe call this `count_response` or `count_request`, or something similar that indicates this is an instrument?